tests/fixtures/node_templates/index_samples/sample_index.ts

```diff
@@ -20,6 +20,7 @@ import * as v1 from './v1';
 import * as v1beta1 from './v1beta1';
 
 const TextToSpeechClient = v1.TextToSpeechClient;
+type TextToSpeechClient = v1.TextToSpeechClient;
```

Comment:
I didn't know this was a thing.

Reply:
Called "declaration merging". Tested [here](https://github.com/googleapis/nodejs-speech/pull/643), fixes [this](https://github.com/googleapis/gapic-generator-typescript/issues/694).